pkg/cli/install.go

```diff
@@ -83,22 +84,24 @@ func (i *Install) Run(cmd *cobra.Command, args []string) error {
 		image = i.Image
 	}
 
-	annotations := map[string]string{}
-	for _, anno := range i.ControllerServiceAccountAnnotation {
-		k, v, ok := strings.Cut(anno, "=")
-		if !ok {
-			return fmt.Errorf("--controller-service-account-annotation must be in key=value format got [%s]", anno)
-		}
-		annotations[k] = v
+	controllerAnnotations, err := parseAnnotations(i.ControllerServiceAccountAnnotation)
```

Comment:
Maybe `controllerSAAnnotations` for clarity?

Reply:
done